experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl

```diff
@@ -365,12 +365,12 @@ end
 
 
 @doc raw"""
-    associated_literature_models(m::AbstractFTheoryModel)
+    birational_literature_models(m::AbstractFTheoryModel)
 
-Return a list of the unique identifiers any `associated_literature_models` of
+Return a list of the unique identifiers any `birational_literature_models` of
```

Comment:
Maybe I am reading it wrong, but the sentence does not make sense to me. Is it missing a word or two?

Reply:
Maybe it should be "... OF any..."?